qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageAppointmentsPage.java

```diff
@@ -15,14 +15,14 @@
 public class ManageAppointmentsPage extends Page {
 
 	public static final By DELETE_REQUEST = By.cssSelector(".delete-item.icon-remove");
-	private static final By CANCEL_BUTTON = By.cssSelector("#searchButtons > button");
-	private static final By SEARCH = By.cssSelector("#searchButtons > button.confirm");
 	private static final By APPOINTMENT = By.xpath("//table[@id='appointmentTable']/div[2]/div/div/div/div[2]/div[2]/div");
-	private static final By NEXT = By.cssSelector("#selectAppointment > button.confirm");
 	private static final By SAVE = By.xpath("//div[@id='confirmAppointment']/div[2]/button[2]");
+	private static final By CANCEL_BUTTON = By.cssSelector("#searchButtons button");
+	private static final By SEARCH = By.cssSelector("#searchButtons button.confirm");
+	private static final By NEXT = By.cssSelector("#selectAppointment button.confirm");
```

Comment:
is `NEXT` a button or a link?

Reply:
> is `NEXT` a button or a link?

@kdaud it is a button

Reply:
how can one know it's a button?

Reply:
Giving it a distinctive name like `NEXT_BUTTON` would be better!

Reply:
How about `CONFIRM_BUTTON`

Reply:
> Why are we combining all these in one class, contrary to the original design which had separated them into different classes?

@dkayiwa I have not understood this question. Do you mean combing the different test into one? If yes, we wanted to create an E2E workflow, hence combining the different test classes.